packages/playground/css/postcss.config.js

```diff
@@ -1,3 +1,12 @@
-module.exports = {
-  plugins: [require('postcss-nested')]
+module.exports = (api) => {
+  if (/\.sss$/.test(api.file)) {
+    return {
```

Comment:
**nitpick (non-blocking):** I would prefer an initial object at the start of the function body and return it at the end of the function
Then in between you can add dynamically properties and extend it

So something like

```ts
  const config = {
    plugins: [require('postcss-nested')]
  }

  if (/\.sss$/.test(api.file)) {
    config.parser = 'sugarss'
    config.plugins.unshift(require('postcss-simple-vars'))
  }

  return config
```

May need to be extended with some type-safety or so